data-canary/scripts/globalevents/startup.lua

```diff
@@ -75,11 +75,6 @@ function startup.onStartup()
 		Spdlog.info("Events: " .. "Exp: " .. expRate .. "%, " .. "loot: " .. lootRate .. "%, " .. "Spawn: " .. spawnRate .. "%, " .. "Skill: ".. skillRate .."%")
 		end
 	end
-
-    -- Client XP Display Mode
-	-- 0 = ignore exp rate /stage
-	-- 1 = include exp rate / stage
-	Game.setStorageValue(GlobalStorage.XpDisplayMode, 1)
 end
 
```

Comment:
isnt your canary considering the hireling as your data-otservbr-global?
noticed cuz im using data-canary exclusively and everything added to data-otbr ive done adding to data-canary.. it would be a bit messy if i were to import direct to my repo (also considering people that uses data-canary instead)

Reply:
Changes were made to both canary and rlmap.

Reply:
'Twasnt what i meant but, well, ist working.
You can set it as resolved.